machine.go

```diff
@@ -449,7 +449,16 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 	}
 	w.CopyFile(prefix + "/lib/x86_64-linux-gnu/libresolv.so.2")
 	w.CopyFile(prefix + "/lib/x86_64-linux-gnu/libc.so.6")
-	w.CopyFile(prefix + "/bin/busybox")
+
+	// search for busybox; in some distros it's located under /sbin
+	busybox, err := exec.LookPath("busybox")
+	if err != nil {
+		busybox, err = exec.LookPath(prefix + "/sbin/busybox")
```

Comment:
As this is teh second workaround for /sbin:/usr/sbin not in the PATH would it not make more sense to expand the path with those ?